docs/reference/ingest/processors/common-options.asciidoc

```diff
@@ -1,5 +1,5 @@
-| `if`             | no       | -        | Conditionally execute this processor.
-| `on_failure`     | no       | -        | Handle failures for this processor. See <<handling-failure-in-pipelines>>.
-| `ignore_failure` | no       | `false`  | Ignore failures for this processor. See <<handling-failure-in-pipelines>>.
-| `tag`            | no       | -        | An identifier for this processor. Useful for debugging and metrics.
-// TODO: See <<ingest-conditionals>>. <-- for the if description once PR 35044 is merged
\ No newline at end of file
+| `description`     | no      | -        | Description of the processor configuration.
```

Comment:
Minor, but the processor description can be anything that's useful to the user and doesn't necessarily have to be about the processor's configuration.

Reply:
I'll change this to:

> Description of the processor. Useful for describing the purpose of the processor or its configuration.